app/javascript/components/__tests__/ProductivityCheckLow.test.js

```diff
@@ -22,15 +22,15 @@ const mockService = {
     test('checking components for crashing', () => {
       render(
         <MemoryRouter>
-            <ProductivityCheckLow data={data} setData={setData} service={mockService} />
+            <ProductivityCheckLow data={data} setData={setData} saveDataToDb={() => {}} steps={[]} service={mockService} draft={false} />
         </MemoryRouter>
```

Comment:
Is there a way to assign this block into some sort of variable that you can then reference in the subsequent tests?